apps/cli/src/prebuild.ts

```diff
@@ -139,21 +175,53 @@ export const prebuild = async () => {
     }
   }
 
-  const cssText = generateCssText(
-    {
-      assets: siteData.assets,
-      breakpoints: siteData.build?.breakpoints,
-      styles: siteData.build?.styles,
-      styleSourceSelections: siteData.build?.styleSourceSelections,
-      componentMetas,
-    },
-    {
-      assetBaseUrl: ASSETS_BASE,
+  const assetsToDownload: Promise<void>[] = [];
+  const fontAssets: FontAsset[] = [];
+  const assetBuildUrl = `https://${domain}.wstd.io/cgi/asset/`;
+
+  for (const asset of siteData.assets) {
+    if (asset.type === "image") {
+      const image = getImageAttributes({
+        /*
+          TODO:
+          https://github.com/webstudio-is/webstudio-builder/issues/2135
+          There should be a option in the loader, that allows to download
+          original image instead of the processed one. Right now, we are using
+          the width from the asset meta
+        */
```

Comment:
The thing about loader is that you can provide own one. Here you just don't want any of its logic. Why complicate it with more options?